src/ui/handler/scroll_zoom.js

```diff
@@ -398,6 +398,7 @@ class ScrollZoomHandler {
     }
 
     _isFullscreen(): boolean {
+        if (isSafari(window)) return !!window.document.webkitFullscreenElement;
```

Comment:
nit: This looks like it should work fine, but it might be a tiny bit cleaner to remove `isSafari(...)` and just always check for `webkitFullScreenElement`:
 `return !!window.document.fullscreenElement || !!window.document.webkitFullscreenElement`.